### ISSUE TITLE
Radiant 1.11.4 unresponsive scrolling with > 95% CPU  usage

### ISSUE THREAD

**Issue by aleksarias** [radiant-player/radiant-player-mac#649]
Installed the latest radiant player and made sure I'm using the NPAPI version of flash. 
When enter the program and scroll, the program freezes and CPU utilization goes to over 95% on a single core.
Here's my OS information:
  ```
Model Name:	MacBook Pro
  Model Identifier:	MacBookPro13,2
  Processor Name:	Intel Core i5
  Processor Speed:	2.9 GHz
  Number of Processors:	1
  Total Number of Cores:	2
  L2 Cache (per Core):	256 KB
  L3 Cache:	4 MB
  Memory:	8 GB
  System Version:	macOS 10.12.3 (16D32)
  Kernel Version:	Darwin 16.4.0
  Boot Volume:	Macintosh HD
  Boot Mode:	Normal
```

Here's a screenshot of the CPU usage while simply scrolling and not playing music.
![untitled](https://cloud.githubusercontent.com/assets/8338500/24175020/605fcd8e-0e6a-11e7-880e-643b369f6af3.jpeg)

**Comment by jacobwgillespie** [comment]
This should be resolved with the upcoming Radiant 2.0 (which doesn't use Flash).

**Comment by Evertt** [comment]
@jacobwgillespie when is 2.0 expected to arrive?

**Comment by jacobwgillespie** [comment]
@Evertt I'm not entirely sure - you can follow along with the progress at https://github.com/radiant-player/radiant-player-electron.  But since it's essentially only my limited free time that is being devoted to development, I'm not sure when I'll have time to finish everything up.  The bit features that are still missing are:

- A settings dialog
- Themes
- Last.fm scrobbling

Then I have to get the pipeline set up to actually release it.

**Comment by chrismou** [comment]
@jacobwgillespie is it still only building for Mac? I had a look at the repo a few months ago but was depending on some Mac only packages

**Comment by jacobwgillespie** [comment]
@chrismou yes, for now.  Technically the only mac-specific thing is the `osx-mouse` package that's being used to access the mouse and thus make the custom title bar work.  There's an equivalent package for Windows that can be used there as a replacement.  I've mainly just been targeting feature-parity with Radiant 1.0 to get this out the door faster, but after that's done then going back and adding Windows support should definitely be doable.

**Comment by chrismou** [comment]
Yeah, I'm getting flashbacks of this being where i got stuck last time. Us pesky Linux users ;-)

Haven't spotted any equivalent Linux packages on your travels have you? I can't actually remember what that OSX mouse package actually does.

**Comment by jacobwgillespie** [comment]
Ah, you're on linux - I haven't seen the equivalent package for Linux, however [the C code](https://github.com/kapetan/osx-mouse/blob/master/source/mouse.cc) is acutally pretty straightforward, so creating an equivalent linux package should be doable, I would imagine.  Or we could just disable the custom menu bar on Linux.
